.travis.yml

```diff
@@ -35,12 +35,16 @@ before_script:
 script:
   - |
     export COVERAGE_PROCESS_START="${TRAVIS_BUILD_DIR}/.coveragerc"
+    export COVERAGE_FILE="${TRAVIS_BUILD_DIR}/.coverage"
+    coverage run -m isodatetime.tests.test_00
+    coverage run -a -m isodatetime.tests.test_datetimeoper
+    coverage run -a -m isodatetime.tests.test_main
     # NOTE: the way to handle long-living commands in the documentation is
-    # travis_wait 40 coverage run -m isodatetime.tests
+    # travis_wait 40 coverage run -m isodatetime.tests.test_time_point
     # but it does not produce any output, which can be frustrating in case of errors.
     # So here we are using the approach documented in the issue
     # https://github.com/travis-ci/travis-ci/issues/4190.
-    coverage run -m isodatetime.tests &
+    coverage run -a -m isodatetime.tests.test_time_point &
```

Comment:
Maybe we could centralize the tests in `setup.py, using just `setuptools` + `pytest`, and skip the slow tests with a `pytest` mark. But enable these tests in Travis-CI?

This way other devs should be able to run the tests in command line with one command, or with tools integration (e.g. tox, IDE's, etc).

Example: https://github.com/matthewrmshin/isodatetime/pull/1

Still need to wait for Travis, and fix if the build takes too long (longer than before) there.